mlflow/ml-package-versions.yml

```diff
@@ -66,66 +66,33 @@ tensorflow:
       pip install tf-nightly
 
   models:
-    minimum: "2.3.0"
+    minimum: "2.0.0"
     maximum: "2.9.1"
     requirements:
-      "< 2.6.0": ["protobuf<4.0.0"]
       # Requirements to run tests for keras
-      "== dev": ["scikit-learn", "pyspark", "pyarrow"]
+      ">= 0.0.0": ["scikit-learn", "pyspark", "pyarrow"]
     run: |
-      pytest tests/tensorflow/test_tensorflow2_model_export.py tests/tensorflow/test_tensorflow2_metric_value_conversion_utils.py
-
-      # Run tests for keras against keras-nightly that's installed with tf-nightly:
-      # https://github.com/tensorflow/tensorflow/blob/v2.6.0/tensorflow/tools/pip_package/setup.py#L110-L122
-      if [ "$PACKAGE_VERSION" == "dev" ]; then
-        pytest tests/keras/test_keras_model_export.py
-      fi
+      tf_version=$(pip freeze | grep 'tensorflow==' | grep -Eo '[0-9].[0-9]')
```

Comment:
TF < 2.5 doesn't install keras. We need to install it.

Reply:
Q: In mlflow 2.0, For TF <= 2.5, if we don't install keras package, we can still use TF embedded keras by `tensorflow.keras`. Shall we use it or use the separate keras ? @dbczumar 
`

Reply:
I think we should use `tensorflow.keras`.

Reply:
OK if so we don't need to install keras package

Reply:
+1